mapsims/cmb/__init__.py

```diff
@@ -0,0 +1,83 @@
+import healpy as hp, numpy as np
+import so_pysm_models as spm
+
+
+def get_cmb_sky(
+    iteration_num,
+    nside=None,  # set this if healpix is desired
+    shape=None,  # set shape and wcs if CAR maps are desired
+    wcs=None,
+    lensed=True,
+    aberrated=False,
+    modulation=False,
+    has_polarization=True,
+    cmb_set=0,  # We allow for more than one CMB map per lensing map
+    cmb_dir=None,
+    nu=None,
+    input_units="uK_RJ",
+    pixel_indices=None,
+):
+    """
+    Return a CMB map from stored alm's.  This can be in Healpix format
+    (if nside is specified) or rectangular pixel format (if wcs and shape are
+    specified).  The lensed alm's are pre-stored.
+    If rectangular, it returns a stack of enmaps of shape (nfreqs, ncomp, ny, nx). 
+    If Healpix, it will return a numpy array of shape (nfreqs, ncomp, npix) 
+
+    Args:
+        iteration_num: integer specifying which sim iteration to use
```

Comment:
can you turn this into using `numpydoc` style arguments like `so_pysm_models`?

Reply:
Pushed.